spec/models/indeed_api_spec.rb

```diff
@@ -11,8 +11,8 @@
         headers: {'Content-Type' => 'application/json'}
       )
 
-    response = IndeedApi.search_jobs(search_params)
+    response = IndeedApi.search_jobs(search_params).first
```

Comment:
Think this is refusing,  you are taking the first output which means this variable is not the entire response.  Might be better to change the var name to something to indicate that it does not have all of the payload attributes.